src/double_integrator/value_iteration.py

```diff
@@ -109,6 +112,26 @@ def draw(iteration, mesh, cost_to_go, policy):
 surf = ax2.plot_surface(Q, Qdot, Pi, rstride=1, cstride=1,
                         cmap=cm.jet)
 
-plt.show()
+# animate the resulting policy.
+builder = DiagramBuilder()
+plant = builder.AddSystem(DoubleIntegrator())
+logger = builder.AddSystem(SignalLogger(2))
+builder.Connect(plant.get_output_port(0), logger.get_input_port(0))
+vi_policy = builder.AddSystem(policy)
+builder.Connect(vi_policy.get_output_port(0), plant.get_input_port(0))
+builder.Connect(plant.get_output_port(0), vi_policy.get_input_port(0))
+
+diagram = builder.Build()
+simulator = Simulator(diagram)
+simulator.set_publish_every_time_step(False)
```

Comment:
you can zap this line (it's no longer needed)

Reply:
Done.